src/runtime/server/handler.ts

```diff
@@ -1,5 +1,4 @@
 import { createError, defineEventHandler, getRequestHeader, getRouterParam, readBody } from 'h3'
```

Comment:
`setResponseStatus` and `setResponseHeaders` will be auto-imported locally, but break the distributed module. Please add the missing imports. (It's a bit confusing during development)